packages/cardhost/app/helpers/scope-css.js

```diff
@@ -0,0 +1,14 @@
+import { helper } from '@ember/component/helper';
+import scope from 'scope-css';
+
+// Prefixes all CSS rules with the parent class name
+// Requires two params, the CSS and a class name or ID to add to the front of every rule.
+
+export default helper(function scopeCss(params) {
+  if (!params[0] || !params[1]) {
+    return '';
+  }
+
+  // local-hub::my-card becomes my-card
+  return scope(params[0], `.${params[1]}`);
```

Comment:
Should we also do any light validation of the class name that is passed in? I'm thinking about the cases where a developer might blindly pass in a class name that has special characters in it that are not valid in css. We could also just throw an error when an invalid class name is passed in

Reply:
I don't think so, for the same reasons that we don't validate the CSS itself either. Browsers are pretty resilient towards "bad" CSS and we are already leaning on that functionality.